QuoteGeneration/quote_wrapper/tdx_quote/td_ql_logic.cpp

```diff
@@ -1403,12 +1417,25 @@ tee_att_error_t tee_att_config_t::ecdsa_init_quote(sgx_ql_cert_key_type_t certif
             goto CLEANUP;
         }
 
-        if(0 != memcpy_s(&p_qe_target_info->mr_enclave, sizeof(p_qe_target_info->mr_enclave),
-                    &tdqe_report.body.mr_enclave, sizeof(tdqe_report.body.mr_enclave))) {
+        if (0 != memcpy_s(&p_qe_target_info->mr_enclave, sizeof(p_qe_target_info->mr_enclave),
+                &tdqe_report.body.mr_enclave, sizeof(tdqe_report.body.mr_enclave))) {
             refqt_ret = TEE_ATT_ERROR_UNEXPECTED;
             goto CLEANUP;
         }
 
+        if (NULL == m_tdqe_report_body) {
+            m_tdqe_report_body = (sgx_report_body_t*)malloc(sizeof(sgx_report_body_t));
```

Comment:
Are we freeing the memory on error cases?

Reply:
same as upper